src/Mimi.jl

```diff
@@ -4,13 +4,16 @@ using DataStructures
 using DataFrames
 using Distributions
 using NamedArrays
+using JSON 
```

Comment:
If we start to use a package here, we also need to list it in REQUIRE, otherwise they will yell at us when we try to tag the package in METADATA.jl. Just add a line with the name to the REQUIRE file.